ports/lcms/portfile.cmake

```
@@ -23,6 +23,7 @@ vcpkg_configure_cmake(
 vcpkg_install_cmake()
 vcpkg_copy_pdbs()
 vcpkg_fixup_cmake_targets()
+vcpkg_fixup_pkgconfig()
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
```

```suggestion
file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
```
The path should be quoted in cmake files. Please see the [doc ](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/cmake-guidelines.md)here.

---

The double quotes have also been added here.